crypto/err/err.c

```diff
@@ -131,9 +101,7 @@ static ERR_STRING_DATA ERR_str_reasons[] = {
      "called a function you should not call"},
     {ERR_R_PASSED_NULL_PARAMETER, "passed a null parameter"},
     {ERR_R_INTERNAL_ERROR, "internal error"},
-    {ERR_R_DISABLED, "called a function that was disabled at compile-time"},
     {ERR_R_INIT_FAIL, "init fail"},
-    {ERR_R_OPERATION_FAIL, "operation fail"},
```

Comment:
Any reason why you're removing this?

Reply:
I too was kind of surprised that this isn't used anywhere, but it isn't so as part of this cleanup PR I removed it.

Reply:
...  and this.